### ISSUE TITLE
Move vcftobedpe to use MATEID to find BND pairs instead of the SECONDARY tag

### ISSUE THREAD

**Issue by ernfrid** [hall-lab/svtools#129]
See comments at the end of #6.

The new implementation can just look at MATEID to see pairs. The only requirement in terms of determining left breakpoint coordinates and right breakpoint is that if they are on the same chromosome the leftmost breakpoint should occur before the rightmost breakpoint in terms of chromosomal order. We don't believe that for trans-chromosomal BNDs the order matters. In the case where there is a contig ordering in the VCF, it may be beneficial to order along those lines as well.

**Comment by brunogrande** [comment]
I've been trying to convert Manta VCF files to BEDPE format and ran into this very issue. I created a fork of svtools to show the small changes I made to get it to work. See https://github.com/brunogrande/svtools/commit/57f56807d9314ab9307e12ee9223b9ab96a80def. This code was not extensively tested, which is why I'm not submitting a pull request. Still, I thought I would share it for your information. 

When updating the code, I noticed that the `EVENT` info field cannot be depended on in Manta VCF files for pairing break-ends, as it can come up multiple times in what I believe are complex rearrangements involving more than two break-ends. This might be related to #93, but I'm not sure. I found that using the `MATEID` info field and the `ID` column was more reliable.
